tools/mo/unit_tests/moc_tf_fe/check_info_messages_test.py

```diff
@@ -52,7 +52,8 @@ def arg_parse_helper(input_model,
         data_type=None,
         tensorflow_custom_operations_config_update=None,
         compress_to_fp16=compress_to_fp16,
-        extensions=None
+        extensions=None,
+        disable_mmap=False
```

Comment:
looks like a misalignment that in some places we have enable, in others disable.

Reply:
`disable_mmap` is always `False` by default. Discussed that it should be default behaviour of MO

Reply:
I mean the option name `enable_mmap` (API) vs `disable_mmap` (MO)

Reply:
@mbencer, @ilya-lavrenov, I agree with @ilya-lavrenov to have `enable_mmap` to be aligned with runtime API.
Let us have `enalbe_mmap` with default value `True`

Reply:
changed to `enalbe_mmap`